README.md

```diff
@@ -1,2 +1,65 @@
-# hello-microverse
-Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-world-microverse-project
+
+> Hello World Project
+
+
+## Built With
+
+- Major languages :\
+  **HTML5**  \
+  **CSS3**
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [Optional] Please consider updating the fields in this section to add the steps required to get a local copy of your project. This will make your work look more professional.

- [Optional] Please in all future projects, make sure your pull request title is a short description of the changes implemented in the branch. This will make your work look professional. An example of a descriptive PR titlr for this project is *Hello Microverse Project*. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more information.

- [Optional] Please in all future projects, make sure the name of your branch is a feature or an action. This is best practice for GItHub workflow. An example of a good name for a feature branch is *build-hello-microverse*. You can check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches) for more details.